site/src/util/date.js

```diff
@@ -32,7 +32,7 @@ module.exports.timeAgo = function(date) {
 
   if (ago > week) {
     const d = new Date(date)
-    return { needsUpdate: false, string: month(d) + d.getDate() }
```

Comment:
whoooooooooooooooops

Reply:
this is what happens when you don't test your code for at least a month :^)

Reply:
did this not go wrong with, say, october?

it's just been overlooked i guess